src/Congo.WebApi/Models/Products/ProductInputModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Congo.WebApi.Models.Products
+{
+    public class ProductInputModel
```

Comment:
I would personally name this something like InsertProductRequest or ProductInsertRequest

Reply:
Is this supposed to be using the Product entity located in 
src\Congo.WebApi\Data\Models\Product..cs ???

Reply:
No, this is a request model that will be sent from the Blazor project whenever they want to add a new Product hence prefixing it with Request makes sense. Because for example when they add a new product they do not need to enter an Id for it because that is our job, unlike our Product model which does have an Id property